### ISSUE TITLE
Java 12.5: Using final with Inheritance

### ISSUE THREAD

**Issue by AkMo3** [girlscript/winter-of-contributing#2428]
### Description

Add audio, documentation, and video regarding the **Using final with Inheritance**.

____

### Location of File to be added

The files should be added inside the `Using final with Inheritance`  folder which is inside the `Inheritance`  folder present in the   `Java`  main folder.  The documentation file can be in the `.md`/`.py`/`.ipynb` format. Audio should be in `.mp3` format and to be submitted through a Google drive link. Video can be in any format and should be submitted through a Google Drive link.

 **Note:**
 - You are required to comment with `\assign CONTENT_TYPE`, so that other content types can be created at the same time.
 - Failing to comment in the above manner will lead to the removal of assignment from the issue even if the bot assigns.
 - You are required to do a PR within 7 days otherwise you will be unassigned and others will be assigned.
 - Issues will be assigned on a first come first serve basis.

### Domain

Java_Domain

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by AbhinandanAdhikari** [comment]
> /assign documentation

@AkMo3 My Java Bytecode PR has been merged so i can take up this one? The bot automatically unassigned me

**Comment by AkMo3** [comment]
Yes as you still have 5 assigned issues. Please finish them first. If you have merged PR for them, let me know.

**Comment by AbhinandanAdhikari** [comment]
> Yes as you still have 5 assigned issues. Please finish them first. If you have merged PR for them, let me know.

@AkMo3 Yes, all my 5 Pull requests are merged

**Comment by ganaaduvva** [comment]
\assign documentation

**Comment by AkMo3** [comment]
@AbhinandanAdhikari You have been assigned documentation. @ganaaduvva He actually commented so I am assigned him, otherwise you can ask for other comment.
